pkg/transforms/http_test.go

```diff
@@ -187,28 +187,28 @@ func TestHTTPPostPutWithSecrets(t *testing.T) {
 		Name                 string
 		Path                 string
 		HeaderName           string
+		SecretValueKey       string
 		SecretName           string
```

Comment:
Order is important below.
```suggestion
		SecretName           string
		SecretValueKey       string

```